playwright/helpers/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const DEFAULT_DATASET = 'test'
```

Comment:
Maybe this would be an acceptable place to put a unique run ID as well?
And we will want to have the space name be defined in a single place, too.

```suggestion
import {uuid} from '@sanity/uuid'

export const TEST_RUN_ID = uuid()
export const STUDIO_SPACE_NAME = 'test'
export const DEFAULT_DATASET = 'test'
```